src/route/dataset-route.ts

```diff
@@ -191,7 +252,7 @@ apiRoute.get('/:dataset/view', async (req, res) => {
     const page_size_str: string = req.query.page_size || req.body?.page_size;
     const page_number: number = Number.parseInt(page_number_str, 10) || 1;
     const page_size: number = Number.parseInt(page_size_str, 10) || DEFAULT_PAGE_SIZE;
-    const processedCSV = await processCSV(dataset, page_number, page_size);
+    const processedCSV = await processCSVFromDatalake(dataset, page_number, page_size);
```

Comment:
As per earlier comment, rework this to be a streamed response rather than batched through an array in memory.

Reply:
I am prepared to do this in a later PR after I have reworked the backend following the new Data model.  I need to land this for show and tell and thats a bigger bit of work.

Dare I suggest we raise a Bug ticket in Jira and stick it in the next sprint.